README.md

```diff
@@ -451,6 +452,25 @@ const opts = {
 hellosign.embedded.postEditUrl(template_id, opts)
     .then();
 ````
+### Reports
+
+#### Gives you insights into user activity and document status
+
+#### Get a Report
+````javaScript
+const opts = {
+  test_mode: 1,
+  start_date: '09/01/2020',
+  end_date: '09/01/2020',
```

Comment:
For the sake of clarity, I would recommend using a number >12 for the day. As it stands, it's not obvious if this should be in the format `mm/dd` or `dd/mm`. In my suggestion below, I chose to increase the range to show that the leading zero is needed for the day in addition to using the number 13 to clearly indicate this as the day field, rather than month

```suggestion
  start_date: '09/09/2020',
  end_date: '09/13/2020',
```

Reply:
Updated to reflect your suggestion.